src/pytesmo/timedate/julian.py

```diff
@@ -1,244 +1,54 @@
 """
-Created on Tue Apr 02 16:50:34 2013
-
-@author: tm
-
-computes julian date, given month (1..12). day(1..31) and year,
-its inverse (calendar date from julian), and the day
-of the year (doy), assuming it is a leap year.
-
-julday and caldat are adapted from "Numerical Recipes in C', 2nd edition,
-pp. 11
-
-restrictions
-- no error handling implemented
-- works only for years past 1582
-- time not yet supported
-
+Most datetime functions from pytesmo are now in the cadati package
+Website: https://github.com/TUW-GEO/cadati
+To install cadati, run 'pip install cadati'
+This module will be removed in future versions of pytesmo
 """
 
 import numpy as np
 import pandas as pd
 import datetime as dt
 import pytz
 
+import warnings
```

Comment:
julian functions are now in cadata, I guess we can delete the remaining ones at some point?